src/components/parameters.js

```diff
@@ -43,7 +44,7 @@ const useStyles = makeStyles(theme => ({
     }
 }));
 
-const Parameters = () => {
+const Parameters = (props) => {
```

Comment:
Just a naive question here:
- I find it more judicious to add only the useful props rather all of them. Is there a good practice in ReactJS or is it up to the developper ?

Reply:
I think that the way you recommend is best than passing just props because it lets us  declare explicitly our properties so we can know what our component is exactly waiting for.